src/components/tile/tile.test.js

```diff
@@ -255,7 +207,7 @@ context("Tests for Tile component", () => {
       "should check border variant as %s for tile component",
```

Comment:
"should check border variant as %s for **Tile** component",

Reply:
updated